compiler/damlc/tests/src/DA/Test/Repl.hs

```diff
@@ -122,3 +128,37 @@ testConnection damlc scriptDar testDar ledgerPort mbTokenFile mbCaCrt = do
                    , [ "--access-token-file=" <> tokenFile | Just tokenFile <- [mbTokenFile] ]
                    , [ "--cacrt=" <> cacrt | Just cacrt <- [mbCaCrt] ]
                    ]
+
+staticTimeTests :: FilePath -> FilePath -> FilePath -> IO Int -> TestTree
+staticTimeTests damlc scriptDar testDar getSandboxPort = testGroup "static-time"
+    [ testCase "setTime" $ do
+        port <- getSandboxPort
+        testSetTime damlc scriptDar testDar port
+    ]
+
+testSetTime
+    :: FilePath
+    -> FilePath
+    -> FilePath
+    -> Int
+    -> Assertion
+testSetTime damlc scriptDar testDar ledgerPort = do
+    out <- readCreateProcess cp $ unlines
+        [ "import DA.Date"
+        , "import DA.Time"
+        , "setTime (time (date 2000 Feb 2) 0 1 2)"
```

Comment:
Maybe worth calling `getTime` afterwards and asserting on the result?

Reply:
Good idea, I've added such a check